### ISSUE TITLE
Cards do not snap into empty column

### ISSUE THREAD

**Issue by morbificagent** [RestyaPlatform/board#4102]
Hi one more time,

while trying to use restya for a while now a phenomen is annoying me and my team-members. If someone wants to move a card into a empty column it wont snap in.
It can snap into the right and the left column which arent empty but as i said not in the empty one.

I think best is to take a look at the video:

[SnapPlease.zip](https://github.com/RestyaPlatform/board/files/4813607/SnapPlease.zip)

**Comment by rrjanbiah** [comment]
*Internal note*

Looks like a regression. If the height of the card is less, it can quickly snap in. But, if the height is higher, it needs some minor shake (from top to bottom) in the list.

**Comment by morbificagent** [comment]
> But, if the height is higher, it needs some minor shake (from top to bottom) in the list.

Yes with medium-height cards some shaking helps but in some cases the card with its preview is so big that i can shake and shake and i cant get the card inside...
Than i have to make a dummy-entry in the empty column so it gets longer and than i can put the "big card" inside... after that deleting the dummy entry...

No fun
